--- libwmf-0.2.2/src/ipa/ipa/font.h.orig	Wed Feb 20 23:35:46 2002
+++ libwmf-0.2.2/src/ipa/ipa/font.h	Wed Feb 20 23:38:11 2002
@@ -530,6 +530,8 @@
 
 	FT_Set_Transform (face,0,0);
 
+	if (!face)
+		return 0;
 	use_kerning = FT_HAS_KERNING (face);
 
 	previous = 0;
@@ -579,6 +579,9 @@
 
 	FT_Set_Transform (face,0,0);
 
+	if (!face)
+		return 0;
+
 	use_kerning = FT_HAS_KERNING (face);
 
 	previous = 0;
--- libwmf-0.2.2/src/ipa/xgd/font.h.orig	Tue Oct 16 10:11:43 2001
+++ libwmf-0.2.2/src/ipa/xgd/font.h	Wed Feb 20 23:49:50 2002
@@ -119,6 +119,9 @@
 
 	FT_Load_Char (face,draw_text->str[0],FT_LOAD_RENDER);
 
+	if (!face)
+		return;
+
 	pt.x += face->glyph->bitmap_left;
 	pt.y -= face->glyph->bitmap_top;
 
