diff -ur libwmf.orig/Makefile.in libwmf/Makefile.in
--- libwmf.orig/Makefile.in	Wed Apr 28 14:41:18 1999
+++ libwmf/Makefile.in	Mon Jul 19 20:09:36 1999
@@ -107,17 +107,17 @@
 	@(cd xgd1.3 && $(MAKE) clean) || exit 1
 
 installdirs:
-	$(MKINSTALLDIRS) $(bindir) \
-	$(libdir)
+	$(MKINSTALLDIRS) $(DESTDIR)$(bindir) \
+	$(DESTDIR)$(libdir)
 
 install: all installdirs installsub
-	$(INSTALL) $(srcdir)/xwmf $(bindir)/xwmf
-	$(INSTALL) $(srcdir)/wmftogif $(bindir)/wmftogif
-	$(INSTALL_DATA) $(srcdir)/libwmf.a $(libdir)/libwmf.a
-	$(INSTALL_DATA) $(srcdir)/libgdwmf.a $(libdir)/libgdwmf.a
-	$(INSTALL_DATA) $(srcdir)/libXwmf.a $(libdir)/libXwmf.a
-	$(INSTALL_DATA) wmfapi.h $(INCLUDEDIR)/wmfapi.h
-	$(INSTALL_DATA) gdwmfapi.h $(INCLUDEDIR)/gdwmfapi.h
-	$(INSTALL_DATA) Xwmfapi.h $(INCLUDEDIR)/Xwmfapi.h
+	$(INSTALL) $(srcdir)/xwmf $(DESTDIR)$(bindir)/xwmf
+	$(INSTALL) $(srcdir)/wmftogif $(DESTDIR)$(bindir)/wmftogif
+	$(INSTALL_DATA) $(srcdir)/libwmf.a $(DESTDIR)$(libdir)/libwmf.a
+	$(INSTALL_DATA) $(srcdir)/libgdwmf.a $(DESTDIR)$(libdir)/libgdwmf.a
+	$(INSTALL_DATA) $(srcdir)/libXwmf.a $(DESTDIR)$(libdir)/libXwmf.a
+	$(INSTALL_DATA) wmfapi.h $(DESTDIR)$(INCLUDEDIR)/wmfapi.h
+	$(INSTALL_DATA) gdwmfapi.h $(DESTDIR)$(INCLUDEDIR)/gdwmfapi.h
+	$(INSTALL_DATA) Xwmfapi.h $(DESTDIR)$(INCLUDEDIR)/Xwmfapi.h
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -ur libwmf.orig/libdib/Makefile.in libwmf/libdib/Makefile.in
--- libwmf.orig/libdib/Makefile.in	Wed Apr 28 14:41:41 1999
+++ libwmf/libdib/Makefile.in	Mon Jul 19 20:09:23 1999
@@ -54,12 +54,12 @@
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a test_dibtoxpm core
 
 installdirs:
-	$(MKINSTALLDIRS) @libdir@ \
-	$(INCLUDEDIR)
+	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
+	$(DESTDIR)$(INCLUDEDIR)
 
 install: all installdirs
-	$(INSTALL_DATA) dibtoxpm.h $(INCLUDEDIR)/dibtoxpm.h
-	$(INSTALL_DATA) libdib.a @libdir@/libdib.a
+	$(INSTALL_DATA) dibtoxpm.h $(DESTDIR)$(INCLUDEDIR)/dibtoxpm.h
+	$(INSTALL_DATA) libdib.a $(DESTDIR)@libdir@/libdib.a
 
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -ur libwmf.orig/xgd1.3/Makefile.in libwmf/xgd1.3/Makefile.in
--- libwmf.orig/xgd1.3/Makefile.in	Wed Apr 28 14:42:04 1999
+++ libwmf/xgd1.3/Makefile.in	Mon Jul 19 20:09:12 1999
@@ -76,13 +76,13 @@
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o xpmgddemo gddemo giftogd a.out *.a core 
 
 installdirs:
-	$(MKINSTALLDIRS) @libdir@ \
-	$(INCLUDEDIR)
+	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
+	$(DESTDIR)$(INCLUDEDIR)
 
 install: all installdirs
-	$(INSTALL_DATA) xgd.h $(INCLUDEDIR)/xgd.h
-	$(INSTALL_DATA) xgdttf.h $(INCLUDEDIR)/xgdttf.h
-	$(INSTALL_DATA) libxgd.a @libdir@/libxgd.a
+	$(INSTALL_DATA) xgd.h $(DESTDIR)$(INCLUDEDIR)/xgd.h
+	$(INSTALL_DATA) xgdttf.h $(DESTDIR)$(INCLUDEDIR)/xgdttf.h
+	$(INSTALL_DATA) libxgd.a $(DESTDIR)@libdir@/libxgd.a
 
 #realcleansub:
 #	@echo real cleaning oledecod in oledecod
