diff -ruN libwmf/Makefile.in libwmf.shared/Makefile.in
--- libwmf/Makefile.in	Sat Jun  2 22:22:33 2001
+++ libwmf.shared/Makefile.in	Sat Jun  2 22:21:35 2001
@@ -8,7 +8,7 @@
 
 DOGD = @GD_LIBS@
 
-LDFLAGS = -L. @LDFLAGS@
+LDFLAGS = @LDFLAGS@
 
 CC = @CC@
 
@@ -39,60 +39,55 @@
 .SUFFIXES: .C .o
 
 WMFLIBOBJECTS =	\
-	wmfapi.o \
-	stack.o \
-	bytelevel.o
+	wmfapi.lo \
+	stack.lo \
+	bytelevel.lo
 
 GDLIBOBJECTS =	\
-	gdwmfapi.o \
-	wmftt.o
+	gdwmfapi.lo \
+	wmftt.lo
 
 XFLIBOBJECTS =	\
-	xfwmfapi.o 
+	xfwmfapi.lo 
 
 EPSLIBOBJECTS =	\
-	xfwmfapi.o 
+	xfwmfapi.lo 
 
 XLIBOBJECTS =	\
-	Xwmfapi.o \
-	Xtext.o \
-	rotated.o
+	Xwmfapi.lo \
+	Xtext.lo \
+	rotated.lo
 
 TARGETS = wmftopng wmftofig wmftoeps xwmf
 
 all: $(TARGETS)
 
 xwmf: libwmf.a libXwmf.a xwmf.o LIBDIB
-	$(CC) $(LDFLAGS) -o xwmf xwmf.o -L. -lXwmf -lwmf $(LIBS)
+	$(CC) $(LDFLAGS) -o xwmf xwmf.o -L./.libs -lXwmf -lwmf -L./libdib/.libs $(LIBS)
 
 wmftopng:	libwmf.a libgdwmf.a wmftopng.o LIBXGD LIBDIB
-	$(CC) $(LDFLAGS) -o wmftopng wmftopng.o -L. -lgdwmf -lwmf $(GDLIBS)
+	$(CC) $(LDFLAGS) -o wmftopng wmftopng.o -L./.libs -lgdwmf -lwmf -L./libdib/.libs -Lxgd-1.7.3/.libs $(GDLIBS)
 
 wmftofig: libwmf.a libxfwmf.a wmftofig.o LIBXFIG
-	$(CC) $(LDFLAGS) -o wmftofig wmftofig.o -L. -lxfwmf -lwmf $(XFLIBS)
+	$(CC) $(LDFLAGS) -o wmftofig wmftofig.o -L./.libs -lxfwmf -lwmf -L./libdib/.libs -L./libxfig/.libs $(XFLIBS)
 
 wmftoeps: libwmf.a libepswmf.a wmftoeps.o LIBXFIG
-	$(CC) $(LDFLAGS) -o wmftoeps wmftoeps.o -L. -lepswmf -lwmf $(EPSLIBS)
+	$(CC) $(LDFLAGS) -o wmftoeps wmftoeps.o -L./.libs -lepswmf -lwmf -L./libdib/.libs -L./libxfig/.libs  $(EPSLIBS)
 
-libwmf.a:	$(WMFLIBOBJECTS)
-	 $(AR) rc libwmf.a $(WMFLIBOBJECTS) 
-	 $(RANLIB) libwmf.a 
-
-libgdwmf.a:	$(GDLIBOBJECTS)
-	 $(AR) rc libgdwmf.a $(GDLIBOBJECTS) 
-	 $(RANLIB) libgdwmf.a
-
-libxfwmf.a:	$(XFLIBOBJECTS)
-	 $(AR) rc libxfwmf.a $(XFLIBOBJECTS) 
-	 $(RANLIB) libxfwmf.a
-
-libepswmf.a:	$(EPSLIBOBJECTS)
-	 $(AR) rc libepswmf.a $(EPSLIBOBJECTS) 
-	 $(RANLIB) libepswmf.a
-
-libXwmf.a:	$(XLIBOBJECTS)
-	 $(AR) rc libXwmf.a $(XLIBOBJECTS) 
-	 $(RANLIB) libXwmf.a
+libwmf.a:	$(WMFLIBOBJECTS) LIBDIB
+	libtool --mode=link $(CC) $(LDFLAGS) -o libwmf.la $(WMFLIBOBJECTS) $(LIBS) -rpath @libdir@
+
+libgdwmf.a:	$(GDLIBOBJECTS) LIBDIB LIBXGD libwmf.a
+	libtool --mode=link $(CC) $(LDFLAGS) -o libgdwmf.la $(GDLIBOBJECTS) -L./ -lwmf $(GDLIBS) -rpath @libdir@
+
+libxfwmf.a:	$(XFLIBOBJECTS) LIBDIB LIBXFIG libwmf.a
+	libtool --mode=link $(CC) $(LDFLAGS) -o libxfwmf.la $(XFLIBOBJECTS) -L./ -lwmf $(XFLIBS) -rpath @libdir@
+
+libepswmf.a:	$(EPSLIBOBJECTS) LIBDIB LIBXFIG libwmf.a
+	libtool --mode=link $(CC) $(LDFLAGS) -o libepswmf.la $(EPSLIBOBJECTS) -L./ -lwmf $(EPSLIBS) -rpath @libdir@
+
+libXwmf.a:	$(XLIBOBJECTS) LIBDIB libwmf.a
+	libtool --mode=link $(CC) $(LDFLAGS) -o libXwmf.la $(XLIBOBJECTS) -L./ -lwmf $(LIBS) -rpath @libdir@
 
 LIBDIB:
 	@echo making libdib in libdib
@@ -109,8 +104,11 @@
 %.o: %.c 
 	$(CC) $(CFLAGS) -c $*.c 
 
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 clean: cleansub
-	rm -rf *.o wmftopng wmftofig wmftoeps xwmf a.out core *.a .pure
+	rm -rf *.o wmftopng wmftofig xwmf a.out core *.a *.la *.so* *.lo .pure
 
 cleansub:
 	@echo cleaning libdib in libdib
@@ -131,7 +129,7 @@
 distclean: realclean
 
 realclean: realcleansub
-	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o wmftopng wmftofig wmftoeps xwmf a.out *.a core .pure
+	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o wmftopng wmftofig xwmf a.out *.a *.la *.lo *.so* core .pure
 
 realcleansub:
 	@echo cleaning libdib in libdib
@@ -145,16 +143,16 @@
 	$(MKINSTALLDIRS) $(DESTDIR)$(bindir) \
 	$(DESTDIR)$(libdir)
 
-install: all installdirs installsub
+install: installdirs installsub
 	$(INSTALL) $(srcdir)/xwmf $(DESTDIR)$(bindir)/xwmf
 	$(INSTALL) $(srcdir)/wmftopng $(DESTDIR)$(bindir)/wmftopng
 	$(INSTALL) $(srcdir)/wmftofig $(DESTDIR)$(bindir)/wmftofig
 	$(INSTALL) $(srcdir)/wmftoeps $(DESTDIR)$(bindir)/wmftoeps
-	$(INSTALL_DATA) $(srcdir)/libwmf.a $(DESTDIR)$(libdir)/libwmf.a
-	$(INSTALL_DATA) $(srcdir)/libgdwmf.a $(DESTDIR)$(libdir)/libgdwmf.a
-	$(INSTALL_DATA) $(srcdir)/libXwmf.a $(DESTDIR)$(libdir)/libXwmf.a
-	$(INSTALL_DATA) $(srcdir)/libxfwmf.a $(DESTDIR)$(libdir)/libxfwmf.a
-	$(INSTALL_DATA) $(srcdir)/libepswmf.a $(DESTDIR)$(libdir)/libepswmf.a
+	libtool $(INSTALL_DATA) $(srcdir)/libwmf.la $(DESTDIR)$(libdir)/libwmf.la
+	libtool $(INSTALL_DATA) $(srcdir)/libgdwmf.la $(DESTDIR)$(libdir)/libgdwmf.la
+	libtool $(INSTALL_DATA) $(srcdir)/libXwmf.la $(DESTDIR)$(libdir)/libXwmf.la
+	libtool $(INSTALL_DATA) $(srcdir)/libxfwmf.la $(DESTDIR)$(libdir)/libxfwmf.la
+	libtool $(INSTALL_DATA) $(srcdir)/libepswmf.la $(DESTDIR)$(libdir)/libepswmf.la
 	$(INSTALL_DATA) wmfapi.h $(DESTDIR)$(INCLUDEDIR)/wmfapi.h
 	$(INSTALL_DATA) gdwmfapi.h $(DESTDIR)$(INCLUDEDIR)/gdwmfapi.h
 	$(INSTALL_DATA) Xwmfapi.h $(DESTDIR)$(INCLUDEDIR)/Xwmfapi.h
diff -ruN libwmf/libdib/Makefile.in libwmf.shared/libdib/Makefile.in
--- libwmf/libdib/Makefile.in	Sat Jun  2 22:22:33 2001
+++ libwmf.shared/libdib/Makefile.in	Sat Jun  2 22:21:46 2001
@@ -1,6 +1,6 @@
 LIBS = @LIBS@
 
-LDFLAGS = @LDFLAGS@ -L.
+LDFLAGS = @LDFLAGS@
 
 CC = @CC@
 
@@ -31,24 +31,26 @@
 .SUFFIXES: .C .o
 
 MYOBJECTS = \
-    ../bytelevel.o \
-    bintree.o \
-    dibtoxpm.o
+    ../bytelevel.lo \
+    bintree.lo \
+    dibtoxpm.lo
 
 all: libdib.a test_dibtoxpm
 
 libdib.a:  $(MYOBJECTS)
-	$(AR) rc libdib.a $(MYOBJECTS)
-	$(RANLIB) libdib.a 
+	libtool --mode=link $(CC) $(LDFLAGS) -o libdib.la $(MYOBJECTS) -rpath @libdir@
 
 test_dibtoxpm:  libdib.a test_dibtoxpm.o
-	$(CC) $(LDFLAGS) -o test_dibtoxpm test_dibtoxpm.o -ldib
+	$(CC) $(LDFLAGS) -o test_dibtoxpm test_dibtoxpm.o -L./.libs -ldib
 
 %.o: %.c
 	$(CC) $(CFLAGS) -c $*.c
 
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 clean:
-	rm -rf *.o *.a test_dibtoxpm a.out core
+	rm -rf *.o *.a *.lo *.la *.so test_dibtoxpm a.out core
 
 realclean: 
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a test_dibtoxpm core
@@ -57,9 +59,8 @@
 	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
 	$(DESTDIR)$(INCLUDEDIR)
 
-install: all installdirs
+install: installdirs
 	$(INSTALL_DATA) dibtoxpm.h $(DESTDIR)$(INCLUDEDIR)/dibtoxpm.h
-	$(INSTALL_DATA) libdib.a $(DESTDIR)@libdir@/libdib.a
-
+	libtool $(INSTALL_DATA) libdib.la $(DESTDIR)@libdir@/libdib.la
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -ruN libwmf/libsvg/Makefile.in libwmf.shared/libsvg/Makefile.in
--- libwmf/libsvg/Makefile.in	Sat Jun  2 22:22:33 2001
+++ libwmf.shared/libsvg/Makefile.in	Sat Jun  2 22:21:54 2001
@@ -50,7 +50,7 @@
 realclean: 
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a core
 
-install: all installdirs
+install: installdirs
 	$(INSTALL_DATA) libsvg.a $(DESTDIR)@libdir@/libsvg.a
 
 installdirs:
diff -ruN libwmf/libxfig/Makefile.in libwmf.shared/libxfig/Makefile.in
--- libwmf/libxfig/Makefile.in	Sat Jun  2 22:22:33 2001
+++ libwmf.shared/libxfig/Makefile.in	Sat Jun  2 22:22:00 2001
@@ -1,6 +1,6 @@
 LIBS = @LIBS@
 
-LDFLAGS = @LDFLAGS@ -L.
+LDFLAGS = @LDFLAGS@
 
 CC = @CC@
 
@@ -31,38 +31,39 @@
 .SUFFIXES: .C .o
 
 XFOBJECTS = \
-	color.o \
-	objlist.o \
-	fileops.o
+	color.lo \
+	objlist.lo \
+	fileops.lo
 
 EPSOBJECTS = \
-	color.o \
-	objlist.o \
-	epsfileops.o
+	color.lo \
+	objlist.lo \
+	epsfileops.lo
 
 
 all: libxfig.a libeps.a
 
 libxfig.a:  $(XFOBJECTS)
-	$(AR) rc libxfig.a $(XFOBJECTS)
-	$(RANLIB) libxfig.a 
+	libtool --mode=link $(CC) $(LDFLAGS) -o libxfig.la $(XFOBJECTS) -rpath @libdir@
 
 libeps.a:  $(EPSOBJECTS)
-	$(AR) rc libeps.a $(EPSOBJECTS)
-	$(RANLIB) libeps.a 
+	libtool --mode=link $(CC) $(LDFLAGS) -o libeps.la $(EPSOBJECTS) -rpath @libdir@
 
 %.o: %.c
 	$(CC) $(CFLAGS) -c $*.c
 
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 clean:
-	rm -rf *.o *.a core
+	rm -rf *.o *.a *.la *.lo *.so* core
 
-realclean: 
-	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a core
+realclean: clean
+	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c
 
-install: all installdirs
-	$(INSTALL_DATA) libxfig.a $(DESTDIR)@libdir@/libxfig.a
-	$(INSTALL_DATA) libeps.a $(DESTDIR)@libdir@/libeps.a
+install: installdirs
+	libtool $(INSTALL_DATA) libxfig.la $(DESTDIR)@libdir@/libxfig.la
+	libtool $(INSTALL_DATA) libeps.la $(DESTDIR)@libdir@/libeps.la
 
 installdirs:
 	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
diff -ruN libwmf/xgd-1.7.3/Makefile.in libwmf.shared/xgd-1.7.3/Makefile.in
--- libwmf/xgd-1.7.3/Makefile.in	Sat Jun  2 22:22:33 2001
+++ libwmf.shared/xgd-1.7.3/Makefile.in	Sat Jun  2 22:22:09 2001
@@ -53,7 +53,8 @@
 #-L. as this allows the gd library itself to be found.
 #Put -L. first so that old versions of the gd library elsewhere
 #on your system can't cause conflicts while building a new one.
-LIBDIRS=-L. @LDFLAGS@
+LDFLAGS=@LDFLAGS@
+LIBDIRS=-L./.libs $(LDFLAGS)
 
 #Location where libxgd.a should be installed by "make install".
 INSTALL_LIB=@libdir@
@@ -80,10 +81,15 @@
 BIN_PROGRAMS=pngtogd pngtogd2 gdtopng gd2topng gd2copypal gdparttopng webpng
 TEST_PROGRAMS=gdtest gddemo gd2time gdtestttf
 
+%.o: %.c
+	$(CC) $(CFLAGS) -c $*.c
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 all: libxgd.a $(PROGRAMS)
 
-install: libxgd.a $(BIN_PROGRAMS)
-	$(INSTALL) libxgd.a $(DESTDIR)$(INSTALL_LIB)/libxgd.a
+install:
+	libtool $(INSTALL) libxgd.la $(DESTDIR)$(INSTALL_LIB)/libxgd.la
 	$(INSTALL) pngtogd $(DESTDIR)$(INSTALL_BIN)/pngtogd
 	$(INSTALL) pngtogd2 $(DESTDIR)$(INSTALL_BIN)/pngtogd2
 	$(INSTALL) gdtopng $(DESTDIR)$(INSTALL_BIN)/gdtopng
@@ -134,19 +140,17 @@
 gdtestttf: gdtestttf.o libxgd.a
 	$(CC) gdtestttf.o -o gdtestttf	$(LIBDIRS) $(LIBS)
 
-libxgd.a: gd.o gd_gd.o gd_gd2.o gd_io.o gd_io_dp.o gd_io_file.o gd_ss.o \
-	gd_io_ss.o gd_png.o gdxpm.o gdfontt.o gdfonts.o gdfontmb.o gdfontl.o \
-	gdfontg.o gdtables.o gdttf.o gdcache.o gdkanji.o \
-	gd.h gdfontt.h gdfonts.h gdfontmb.h gdfontl.h gdfontg.h
-	rm -f libxgd.a
-	$(AR) rc libxgd.a gd.o gd_gd.o gd_gd2.o gd_io.o gd_io_dp.o \
-		gd_io_file.o gd_ss.o gd_io_ss.o gd_png.o gdxpm.o \
-		gdfontt.o gdfonts.o gdfontmb.o gdfontl.o gdfontg.o \
-		gdtables.o gdttf.o gdcache.o gdkanji.o
-	-@RANLIB@ libxgd.a
+
+XGDOBJECTS = \
+	gd.lo gd_gd.lo gd_gd2.lo gd_io.lo gd_io_dp.lo gd_io_file.lo gd_ss.lo \
+	gd_io_ss.lo gd_png.lo gdxpm.lo gdfontt.lo gdfonts.lo gdfontmb.lo gdfontl.lo \
+	gdfontg.lo gdtables.lo gdttf.lo gdcache.lo gdkanji.lo
+
+libxgd.a: $(XGDOBJECTS) gd.h gdfontt.h gdfonts.h gdfontmb.h gdfontl.h gdfontg.h
+	libtool --mode=link $(CC) -lm -lpng -lz -lttf $(LDFLAGS) -o libxgd.la $(XGDOBJECTS) -rpath @libdir@
 
 clean:
-	rm -f *.o *.a ${PROGRAMS} core 
+	rm -f *.o *.a *.la *.lo *.so* ${PROGRAMS}
 
 realclean:
 	rm -f *.o *.a ${PROGRAMS} core Makefile
