diff -Nur libwmf-0.2.6.orig/src/font.c libwmf-0.2.6/src/font.c
--- libwmf-0.2.6.orig/src/font.c	Wed Jun 12 20:22:02 2002
+++ libwmf-0.2.6/src/font.c	Sun Jul  7 19:09:15 2002
@@ -512,6 +512,9 @@
 
 	FT_Set_Transform (face,0,0);
 
+	if (!face)
+		return 0;
+
 	use_kerning = FT_HAS_KERNING (face);
 
 	previous = 0;
@@ -559,6 +562,9 @@
 
 	FT_Set_Transform (face,0,0);
 
+	if (!face)
+		return 0;
+
 	use_kerning = FT_HAS_KERNING (face);
 
 	previous = 0;
diff -Nur libwmf-0.2.6.orig/src/ipa/x/font.h libwmf-0.2.6/src/ipa/x/font.h
--- libwmf-0.2.6.orig/src/ipa/x/font.h	Tue Oct 16 10:11:46 2001
+++ libwmf-0.2.6/src/ipa/x/font.h	Sun Jul  7 19:07:39 2002
@@ -122,6 +122,9 @@
 
 	FT_Load_Char (face,draw_text->str[0],FT_LOAD_RENDER);
 
+	if (!face)
+		return;
+
 	pt.x += face->glyph->bitmap_left;
 	pt.y -= face->glyph->bitmap_top;
 
diff -Nur libwmf-0.2.6.orig/src/ipa/xgd/font.h libwmf-0.2.6/src/ipa/xgd/font.h
--- libwmf-0.2.6.orig/src/ipa/xgd/font.h	Sun Oct 28 15:07:21 2001
+++ libwmf-0.2.6/src/ipa/xgd/font.h	Sun Jul  7 19:07:17 2002
@@ -119,6 +119,9 @@
 
 	FT_Load_Char (face,draw_text->str[0],FT_LOAD_RENDER);
 
+	if (!face)
+		return;
+
 	pt.x += face->glyph->bitmap_left;
 	pt.y -= face->glyph->bitmap_top;
 
