--- libwmf-0.2.8.4/src/extra/gd/gd_png.c	2005-07-27 22:35:06.000000000 +0200
+++ libwmf-0.2.8.4/src/extra/gd/gd_png.c.new	2010-01-28 18:47:25.381590959 +0100
@@ -136,7 +136,7 @@
   /* first do a quick check that the file really is a PNG image; could
    * have used slightly more general png_sig_cmp() function instead */
   gdGetBuf (sig, 8, infile);
-  if (!png_check_sig (sig, 8))
+  if (png_sig_cmp(sig, 0, 8) != 0)
     return NULL;		/* bad signature */
 
 #ifndef PNG_SETJMP_NOT_SUPPORTED
