diff -Nur libwmf.orig/Makefile.in libwmf/Makefile.in
--- libwmf.orig/Makefile.in	Thu Dec 16 16:31:01 1999
+++ libwmf/Makefile.in	Thu Dec 28 14:32:41 2000
@@ -130,19 +130,19 @@
 	@(cd libxfig && $(MAKE) realclean) || exit 1
 
 installdirs:
-	$(MKINSTALLDIRS) $(bindir) \
-	$(libdir)
+	$(MKINSTALLDIRS) $(DESTDIR)$(bindir) \
+	$(DESTDIR)$(libdir)
 
 install: all installdirs installsub
-	$(INSTALL) $(srcdir)/xwmf $(bindir)/xwmf
-	$(INSTALL) $(srcdir)/wmftopng $(bindir)/wmftopng
-	$(INSTALL) $(srcdir)/wmftofig $(bindir)/wmftofig
-	$(INSTALL_DATA) $(srcdir)/libwmf.a $(libdir)/libwmf.a
-	$(INSTALL_DATA) $(srcdir)/libgdwmf.a $(libdir)/libgdwmf.a
-	$(INSTALL_DATA) $(srcdir)/libXwmf.a $(libdir)/libXwmf.a
-	$(INSTALL_DATA) $(srcdir)/libxfwmf.a $(libdir)/libxfwmf.a
-	$(INSTALL_DATA) wmfapi.h $(INCLUDEDIR)/wmfapi.h
-	$(INSTALL_DATA) gdwmfapi.h $(INCLUDEDIR)/gdwmfapi.h
-	$(INSTALL_DATA) Xwmfapi.h $(INCLUDEDIR)/Xwmfapi.h
+	$(INSTALL) $(srcdir)/xwmf $(DESTDIR)$(bindir)/xwmf
+	$(INSTALL) $(srcdir)/wmftopng $(DESTDIR)$(bindir)/wmftopng
+	$(INSTALL) $(srcdir)/wmftofig $(DESTDIR)$(bindir)/wmftofig
+	$(INSTALL_DATA) $(srcdir)/libwmf.a $(DESTDIR)$(libdir)/libwmf.a
+	$(INSTALL_DATA) $(srcdir)/libgdwmf.a $(DESTDIR)$(libdir)/libgdwmf.a
+	$(INSTALL_DATA) $(srcdir)/libXwmf.a $(DESTDIR)$(libdir)/libXwmf.a
+	$(INSTALL_DATA) $(srcdir)/libxfwmf.a $(DESTDIR)$(libdir)/libxfwmf.a
+	$(INSTALL_DATA) wmfapi.h $(DESTDIR)$(INCLUDEDIR)/wmfapi.h
+	$(INSTALL_DATA) gdwmfapi.h $(DESTDIR)$(INCLUDEDIR)/gdwmfapi.h
+	$(INSTALL_DATA) Xwmfapi.h $(DESTDIR)$(INCLUDEDIR)/Xwmfapi.h
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -Nur libwmf.orig/libdib/Makefile.in libwmf/libdib/Makefile.in
--- libwmf.orig/libdib/Makefile.in	Tue Jul 13 15:14:28 1999
+++ libwmf/libdib/Makefile.in	Thu Dec 28 14:35:01 2000
@@ -54,12 +54,12 @@
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a test_dibtoxpm core
 
 installdirs:
-	$(MKINSTALLDIRS) @libdir@ \
-	$(INCLUDEDIR)
+	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
+	$(DESTDIR)$(INCLUDEDIR)
 
 install: all installdirs
-	$(INSTALL_DATA) dibtoxpm.h $(INCLUDEDIR)/dibtoxpm.h
-	$(INSTALL_DATA) libdib.a @libdir@/libdib.a
+	$(INSTALL_DATA) dibtoxpm.h $(DESTDIR)$(INCLUDEDIR)/dibtoxpm.h
+	$(INSTALL_DATA) libdib.a $(DESTDIR)@libdir@/libdib.a
 
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -Nur libwmf.orig/libsvg/Makefile.in libwmf/libsvg/Makefile.in
--- libwmf.orig/libsvg/Makefile.in	Thu Dec  9 14:07:57 1999
+++ libwmf/libsvg/Makefile.in	Thu Dec 28 14:35:49 2000
@@ -51,11 +51,11 @@
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a core
 
 install: all installdirs
-	$(INSTALL_DATA) libsvg.a @libdir@/libsvg.a
+	$(INSTALL_DATA) libsvg.a $(DESTDIR)@libdir@/libsvg.a
 
 installdirs:
-	$(MKINSTALLDIRS) @libdir@ \
-	$(INCLUDEDIR)
+	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
+	$(DESTDIR)$(INCLUDEDIR)
 
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -Nur libwmf.orig/libxfig/Makefile.in libwmf/libxfig/Makefile.in
--- libwmf.orig/libxfig/Makefile.in	Tue Jul 13 15:16:20 1999
+++ libwmf/libxfig/Makefile.in	Thu Dec 28 14:36:37 2000
@@ -51,11 +51,11 @@
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a core
 
 install: all installdirs
-	$(INSTALL_DATA) libxfig.a @libdir@/libxfig.a
+	$(INSTALL_DATA) libxfig.a $(DESTDIR)@libdir@/libxfig.a
 
 installdirs:
-	$(MKINSTALLDIRS) @libdir@ \
-	$(INCLUDEDIR)
+	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
+	$(DESTDIR)$(INCLUDEDIR)
 
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -Nur libwmf.orig/xgd-1.7.3/Makefile.in libwmf/xgd-1.7.3/Makefile.in
--- libwmf.orig/xgd-1.7.3/Makefile.in	Thu Dec 16 16:31:47 1999
+++ libwmf/xgd-1.7.3/Makefile.in	Thu Dec 28 14:38:50 2000
@@ -56,13 +56,13 @@
 LIBDIRS=-L. 
 
 #Location where libxgd.a should be installed by "make install".
-INSTALL_LIB=@libdir@
+INSTALL_LIB=$(DESTDIR)@libdir@
 
 #Location where .h files should be installed by "make install".
-INSTALL_INCLUDE=@includedir@
+INSTALL_INCLUDE=$(DESTDIR)@includedir@
 
 #Location where useful non-test programs should be installed by "make install".
-INSTALL_BIN=@bindir@
+INSTALL_BIN=$(DESTDIR)@bindir@
 
 #
 #
