diff -urN libwmf.orig/Makefile.in libwmf/Makefile.in
--- libwmf.orig/Makefile.in	Thu Jan 18 17:55:43 2001
+++ libwmf/Makefile.in	Thu Jan 18 17:58:18 2001
@@ -39,24 +39,24 @@
 .SUFFIXES: .C .o
 
 WMFLIBOBJECTS =	\
-	wmfapi.o \
-	stack.o \
-	bytelevel.o
+	wmfapi.lo \
+	stack.lo \
+	bytelevel.lo
 
 GDLIBOBJECTS =	\
-	gdwmfapi.o \
-	wmftt.o
+	gdwmfapi.lo \
+	wmftt.lo
 
 XFLIBOBJECTS =	\
-	xfwmfapi.o 
+	xfwmfapi.lo 
 
 EPSLIBOBJECTS =	\
-	xfwmfapi.o 
+	xfwmfapi.lo 
 
 XLIBOBJECTS =	\
-	Xwmfapi.o \
-	Xtext.o \
-	rotated.o
+	Xwmfapi.lo \
+	Xtext.lo \
+	rotated.lo
 
 TARGETS = wmftopng wmftofig wmftoeps xwmf
 
@@ -75,24 +75,24 @@
 	$(CC) $(LDFLAGS) -o wmftoeps wmftoeps.o -L. -lepswmf -lwmf $(EPSLIBS)
 
 libwmf.a:	$(WMFLIBOBJECTS)
-	 $(AR) rc libwmf.a $(WMFLIBOBJECTS) 
-	 $(RANLIB) libwmf.a 
+	libtool $(CC) $(LDFLAGS) -o libwmf.la $(WMFLIBOBJECTS) -rpath @libdir@
+	mv -f .libs/libwmf.{a,so*} .
 
 libgdwmf.a:	$(GDLIBOBJECTS)
-	 $(AR) rc libgdwmf.a $(GDLIBOBJECTS) 
-	 $(RANLIB) libgdwmf.a
+	libtool $(CC) $(LDFLAGS) -o libgdwmf.la $(GDLIBOBJECTS) -rpath @libdir@
+	mv -f .libs/libgdwmf.{a,so*} .
 
 libxfwmf.a:	$(XFLIBOBJECTS)
-	 $(AR) rc libxfwmf.a $(XFLIBOBJECTS) 
-	 $(RANLIB) libxfwmf.a
+	libtool $(CC) $(LDFLAGS) -o libxfwmf.la $(XFLIBOBJECTS) -rpath @libdir@
+	mv -f .libs/libxfwmf.{a,so*} .
 
 libepswmf.a:	$(EPSLIBOBJECTS)
-	 $(AR) rc libepswmf.a $(EPSLIBOBJECTS) 
-	 $(RANLIB) libepswmf.a
+	libtool $(CC) $(LDFLAGS) -o libepswmf.la $(EPSLIBOBJECTS) -rpath @libdir@
+	mv -f .libs/libepswmf.{a,so*} .
 
 libXwmf.a:	$(XLIBOBJECTS)
-	 $(AR) rc libXwmf.a $(XLIBOBJECTS) 
-	 $(RANLIB) libXwmf.a
+	libtool $(CC) $(LDFLAGS) -o libXwmf.la $(XLIBOBJECTS) -rpath @libdir@
+	mv -f .libs/libXwmf.{a,so*} .
 
 LIBDIB:
 	@echo making libdib in libdib
@@ -109,8 +109,11 @@
 %.o: %.c 
 	$(CC) $(CFLAGS) -c $*.c 
 
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 clean: cleansub
-	rm -rf *.o wmftopng wmftofig wmftoeps xwmf a.out core *.a .pure
+	rm -rf *.o wmftopng wmftofig xwmf a.out core *.a *.la *.so* *.lo .pure
 
 cleansub:
 	@echo cleaning libdib in libdib
@@ -131,7 +134,7 @@
 distclean: realclean
 
 realclean: realcleansub
-	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o wmftopng wmftofig wmftoeps xwmf a.out *.a core .pure
+	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o wmftopng wmftofig xwmf a.out *.a *.la *.lo *.so* core .pure
 
 realcleansub:
 	@echo cleaning libdib in libdib
@@ -155,6 +158,16 @@
 	$(INSTALL_DATA) $(srcdir)/libXwmf.a $(DESTDIR)$(libdir)/libXwmf.a
 	$(INSTALL_DATA) $(srcdir)/libxfwmf.a $(DESTDIR)$(libdir)/libxfwmf.a
 	$(INSTALL_DATA) $(srcdir)/libepswmf.a $(DESTDIR)$(libdir)/libepswmf.a
+	$(INSTALL_DATA) $(srcdir)/libwmf.la $(DESTDIR)$(libdir)/libwmf.la
+	$(INSTALL_DATA) $(srcdir)/libgdwmf.la $(DESTDIR)$(libdir)/libgdwmf.la
+	$(INSTALL_DATA) $(srcdir)/libXwmf.la $(DESTDIR)$(libdir)/libXwmf.la
+	$(INSTALL_DATA) $(srcdir)/libxfwmf.la $(DESTDIR)$(libdir)/libxfwmf.la
+	$(INSTALL_DATA) $(srcdir)/libepswmf.la $(DESTDIR)$(libdir)/libepswmf.la
+	$(INSTALL_DATA) $(srcdir)/libwmf.so.* $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) $(srcdir)/libgdwmf.so.* $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) $(srcdir)/libXwmf.so.* $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) $(srcdir)/libxfwmf.so.* $(DESTDIR)$(libdir)
+	$(INSTALL_DATA) $(srcdir)/libepswmf.so.* $(DESTDIR)$(libdir)
 	$(INSTALL_DATA) wmfapi.h $(DESTDIR)$(INCLUDEDIR)/wmfapi.h
 	$(INSTALL_DATA) gdwmfapi.h $(DESTDIR)$(INCLUDEDIR)/gdwmfapi.h
 	$(INSTALL_DATA) Xwmfapi.h $(DESTDIR)$(INCLUDEDIR)/Xwmfapi.h
diff -urN libwmf.orig/libdib/Makefile.in libwmf/libdib/Makefile.in
--- libwmf.orig/libdib/Makefile.in	Thu Jan 18 17:55:43 2001
+++ libwmf/libdib/Makefile.in	Thu Jan 18 17:56:15 2001
@@ -31,15 +31,15 @@
 .SUFFIXES: .C .o
 
 MYOBJECTS = \
-    ../bytelevel.o \
-    bintree.o \
-    dibtoxpm.o
+    ../bytelevel.lo \
+    bintree.lo \
+    dibtoxpm.lo
 
 all: libdib.a test_dibtoxpm
 
 libdib.a:  $(MYOBJECTS)
-	$(AR) rc libdib.a $(MYOBJECTS)
-	$(RANLIB) libdib.a 
+	libtool $(CC) $(LDFLAGS) -o libdib.la $(MYOBJECTS) -rpath @libdir@
+	mv -f .libs/libdib.{a,so*} .
 
 test_dibtoxpm:  libdib.a test_dibtoxpm.o
 	$(CC) $(LDFLAGS) -o test_dibtoxpm test_dibtoxpm.o -ldib
@@ -47,8 +47,11 @@
 %.o: %.c
 	$(CC) $(CFLAGS) -c $*.c
 
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 clean:
-	rm -rf *.o *.a test_dibtoxpm a.out core
+	rm -rf *.o *.a *.lo *.la *.so test_dibtoxpm a.out core
 
 realclean: 
 	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a test_dibtoxpm core
@@ -60,6 +63,8 @@
 install: all installdirs
 	$(INSTALL_DATA) dibtoxpm.h $(DESTDIR)$(INCLUDEDIR)/dibtoxpm.h
 	$(INSTALL_DATA) libdib.a $(DESTDIR)@libdir@/libdib.a
-
+	$(INSTALL_DATA) libdib.so.* $(DESTDIR)@libdir@
+	cp -df libdib.so $(DESTDIR)@libdir@
+	$(INSTALL_DATA) libdib.la $(DESTDIR)@libdir@/libdib.la
 
 #  DO  NOT  DELETE THIS LINE -- make depend depends on it.
diff -urN libwmf.orig/libxfig/Makefile.in libwmf/libxfig/Makefile.in
--- libwmf.orig/libxfig/Makefile.in	Thu Jan 18 17:55:43 2001
+++ libwmf/libxfig/Makefile.in	Thu Jan 18 17:57:27 2001
@@ -31,38 +31,47 @@
 .SUFFIXES: .C .o
 
 XFOBJECTS = \
-	color.o \
-	objlist.o \
-	fileops.o
+	color.lo \
+	objlist.lo \
+	fileops.lo
 
 EPSOBJECTS = \
-	color.o \
-	objlist.o \
-	epsfileops.o
+	color.lo \
+	objlist.lo \
+	epsfileops.lo
 
 
 all: libxfig.a libeps.a
 
 libxfig.a:  $(XFOBJECTS)
-	$(AR) rc libxfig.a $(XFOBJECTS)
-	$(RANLIB) libxfig.a 
+	libtool $(CC) $(LDFLAGS) -o libxfig.la $(XFOBJECTS) -rpath @libdir@
+	mv -f .libs/libxfig.{a,so*} .
 
 libeps.a:  $(EPSOBJECTS)
-	$(AR) rc libeps.a $(EPSOBJECTS)
-	$(RANLIB) libeps.a 
+	libtool $(CC) $(LDFLAGS) -o libeps.la $(EPSOBJECTS) -rpath @libdir@
+	mv -f .libs/libeps.{a,so*} .
 
 %.o: %.c
 	$(CC) $(CFLAGS) -c $*.c
 
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 clean:
-	rm -rf *.o *.a core
+	rm -rf *.o *.a *.la *.lo *.so* core
 
-realclean: 
-	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c *.o *.a core
+realclean: clean
+	rm -rf config.cache config.status config.log Makefile config.h lex.yy.c
 
 install: all installdirs
 	$(INSTALL_DATA) libxfig.a $(DESTDIR)@libdir@/libxfig.a
+	$(INSTALL_DATA) libxfig.la $(DESTDIR)@libdir@/libxfig.la
+	$(INSTALL_DATA) libxfig.so.* $(DESTDIR)@libdir@
+	cp -df libxfig.so $(DESTDIR)@libdir@
 	$(INSTALL_DATA) libeps.a $(DESTDIR)@libdir@/libeps.a
+	$(INSTALL_DATA) libeps.la $(DESTDIR)@libdir@/libeps.la
+	$(INSTALL_DATA) libeps.so.* $(DESTDIR)@libdir@
+	cp -df libeps.so $(DESTDIR)@libdir@
 
 installdirs:
 	$(MKINSTALLDIRS) $(DESTDIR)@libdir@ \
diff -urN libwmf.orig/xgd-1.7.3/Makefile.in libwmf/xgd-1.7.3/Makefile.in
--- libwmf.orig/xgd-1.7.3/Makefile.in	Thu Jan 18 17:55:43 2001
+++ libwmf/xgd-1.7.3/Makefile.in	Thu Jan 18 17:56:15 2001
@@ -80,10 +80,18 @@
 BIN_PROGRAMS=pngtogd pngtogd2 gdtopng gd2topng gd2copypal gdparttopng webpng
 TEST_PROGRAMS=gdtest gddemo gd2time gdtestttf
 
+%.o: %.c
+	$(CC) $(CFLAGS) -c $*.c
+%.lo: %.c
+	libtool $(CC) $(CFLAGS) -c $*.c
+
 all: libxgd.a $(PROGRAMS)
 
 install: libxgd.a $(BIN_PROGRAMS)
 	$(INSTALL) libxgd.a $(DESTDIR)$(INSTALL_LIB)/libxgd.a
+	$(INSTALL) libxgd.la $(DESTDIR)$(INSTALL_LIB)/libxgd.la
+	$(INSTALL) libxgd.so.* $(DESTDIR)$(INSTALL_LIB)
+	cp -df libxgd.so $(DESTDIR)$(INSTALL_LIB)
 	$(INSTALL) pngtogd $(DESTDIR)$(INSTALL_BIN)/pngtogd
 	$(INSTALL) pngtogd2 $(DESTDIR)$(INSTALL_BIN)/pngtogd2
 	$(INSTALL) gdtopng $(DESTDIR)$(INSTALL_BIN)/gdtopng
@@ -134,19 +142,19 @@
 gdtestttf: gdtestttf.o libxgd.a
 	$(CC) gdtestttf.o -o gdtestttf	$(LIBDIRS) $(LIBS)
 
-libxgd.a: gd.o gd_gd.o gd_gd2.o gd_io.o gd_io_dp.o gd_io_file.o gd_ss.o \
-	gd_io_ss.o gd_png.o gdxpm.o gdfontt.o gdfonts.o gdfontmb.o gdfontl.o \
-	gdfontg.o gdtables.o gdttf.o gdcache.o gdkanji.o \
-	gd.h gdfontt.h gdfonts.h gdfontmb.h gdfontl.h gdfontg.h
+
+XGDOBJECTS = \
+	gd.lo gd_gd.lo gd_gd2.lo gd_io.lo gd_io_dp.lo gd_io_file.lo gd_ss.lo \
+	gd_io_ss.lo gd_png.lo gdxpm.lo gdfontt.lo gdfonts.lo gdfontmb.lo gdfontl.lo \
+	gdfontg.lo gdtables.lo gdttf.lo gdcache.lo gdkanji.lo
+
+libxgd.a: $(XGDOBJECTS) gd.h gdfontt.h gdfonts.h gdfontmb.h gdfontl.h gdfontg.h
 	rm -f libxgd.a
-	$(AR) rc libxgd.a gd.o gd_gd.o gd_gd2.o gd_io.o gd_io_dp.o \
-		gd_io_file.o gd_ss.o gd_io_ss.o gd_png.o gdxpm.o \
-		gdfontt.o gdfonts.o gdfontmb.o gdfontl.o gdfontg.o \
-		gdtables.o gdttf.o gdcache.o gdkanji.o
-	-@RANLIB@ libxgd.a
+	libtool $(CC) -lm -lpng -lz -lttf $(LDFLAGS) -o libxgd.la $(XGDOBJECTS) -rpath @libdir@
+	mv -f .libs/libxgd.{a,so*} .
 
 clean:
-	rm -f *.o *.a ${PROGRAMS} core 
+	rm -f *.o *.a *.la *.lo *.so* ${PROGRAMS}
 
 realclean:
 	rm -f *.o *.a ${PROGRAMS} core Makefile
